helm/chart/Chart.yaml

```diff
@@ -41,7 +41,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.2.3
+version: 0.2.4
```

Comment:
you could update this to `0.2.5` so that the helm chart release PR already has the right version for the next release